src/core/types.py

```diff
@@ -51,6 +51,11 @@ class Azure(KubernetesCluster):
     name = "Azure"
 
 
+class KubeStateMetricsClient():
```

Comment:
What does a client do and is this the correct way to define it under `core/types`?

Reply:
see https://github.com/aquasecurity/kube-hunter/pull/226#discussion_r338573943